src/main/resources/application.properties

```diff
@@ -122,7 +122,7 @@ server.error.include-stacktrace=never
 # PDF generator configuration
 nae.pdf.resources.outputFolder=storage
 nae.pdf.resources.outputDefaultName=generated_pdf.pdf
-nae.pdf.resources.templateResource=file:src/main/resources/pdfGenerator/template_pdf.pdf
+nae.pdf.resources.templateResource=classpath:pdfGenerator/template.pdf
 nae.pdf.resources.font-title-resource=file:src/main/resources/pdfGenerator/fonts/Roboto-Light.ttf
 nae.pdf.resources.font-label-resource=file:src/main/resources/pdfGenerator/fonts/Roboto-Light.ttf
```

Comment:
should the font files not also be referenced with `classpath:` and not via `file:`?
why were the light font files removed, if they are being referenced?

Reply:
Modified to classpath, and Roboto-light was not removed.